addons/website_twitter/static/tests/tours/twitter_scroller_reload_button.js

```diff
@@ -0,0 +1,44 @@
+odoo.define('website_twitter.tour.twitter_scroller_reload_button', function (require) {
+'use strict';
+
+var tour = require('web_tour.tour');
+var base = require('web_editor.base');
+
+tour.register('twitter_scroller_reload_button', {
+    test: true,
+    url: '/',
+    wait_for: base.ready(),
+}, [{
+    trigger: 'a[data-action=edit]',
+    content: 'Click the Edit button.',
+    extra_trigger: '.homepage',
+}, {
+    trigger: '#snippet_feature .oe_snippet:has(span:contains("Twitter Scroller")) .oe_snippet_thumbnail',
+    content: 'Drag the Twitter Scroller snippet and drop it in your page.',
+    run: 'drag_and_drop #wrap',
+}, {
+    trigger: '#wrap .twitter',
+    content: 'Activate the Twitter Scroller snippet editor.',
+}, {
+    trigger: '#wrap .twitter .twitter_timeline center',
+    content: 'Check if the reload button was added to the DOM.',
+    run: function () {
+        if ($('.btn span:contains("Reload")').length === 0) {
+            console.error('The twitter scroller reload button was not added to the DOM');
+        }
```

Comment:
Why is this not a trigger or an extra trigger of a step?

Reply:
mm, good point!

Reply:
If that's a good point, why was this not changed and the conversation marked as resolved?

Reply:
The button is invisible, checking with an `extra_trigger` does not work in this case

Reply:
You can use the `:has` selector of jQuery to solve that problem :+1: